nixos/modules/services/networking/unbound.nix

```diff
@@ -31,19 +29,25 @@ let
 
   confFile = pkgs.writeText "unbound.conf" ''
     server:
+      ip-freebind: yes
       directory: "${stateDir}"
       username: unbound
-      chroot: "${stateDir}"
+      chroot: ""
       pidfile: ""
+      # when running under systemd there is no need to daemonize
+      do-daemonize: no
       ${interfaces}
       ${access}
       ${trustAnchor}
+    ${lib.optionalString (cfg.localControlSocketPath != null) ''
+      remote-control:
+        control-enable: yes
+        control-interface: ${cfg.localControlSocketPath}
+    ''}
     ${cfg.extraConfig}
```

Comment:
Introducing `cfg.localControlSocketPath` made impossible to write `do`-options (like `do-ip6: no`) in `cfg.extraConfig`, because those options must be BEFORE `remote-control`

Reply:
You can just begin a new `server:` block in `extraConfig`, at least I used to run a config like that.